src/Matrix.jl

```diff
@@ -1923,7 +1923,7 @@ end
 Return an array consisting of k-combinations of {1,...,n} (or a given vector v)
 as arrays.
 """
-function combinations(n::Int, k::Int) combinations(1:n, k) end
+combinations(n::Int, k::Int) = combinations(1:n, k)
```

Comment:
This is adding the docstring to the one method.

One idea is to do
```julia
@doc Markdown.doc"""
    combinations(n::Int, k::Int)
    combinations(v::AbstractVector, k::Int)

Return an array consisting of k-combinations of {1,...,n} (or a given vector v)
as arrays.
"""
function combinations end
```

```

Reply:
Right, thanks! I think I'll just add docstrings for both methods.